rebar.config

```diff
@@ -87,3 +89,8 @@
 %% Disable warning_as_errors for redbug to avoid deprecation warnings.
 {overrides, [ {del, redbug, [{erl_opts, [warnings_as_errors]}]}
             ]}.
+
+{erlfmt, [
+    write,
+    {files, "apps/*/{src,include,test}/*.{erl,hrl,app.src}"}
```

Comment:
```suggestion
    {files, ["apps/*/{src,include,test}/*.{erl,hrl,app.src}", "rebar.config", "elvis.config"]},
```

Reply:
Right, I planned to do it but then forgot. Thanks.